app/src/main/java/de/mytfg/app/android/MyTFG.java

```diff
@@ -56,7 +57,7 @@ public static String getDeviceId() {
     public static String getDate(long timestamp) {
         Calendar calendar = Calendar.getInstance(Locale.ENGLISH);
         calendar.setTimeInMillis(timestamp * 1000);
-        return DateFormat.format("dd. MM. yyyy, HH:mm", calendar).toString() + "h";
+        return DateFormat.format("dd.MM.yyyy, HH:mm", calendar).toString();
```

Comment:
Wozu Leerzeichen? Und Einheiten sollte man vlt. wenn überhaupt in die strings.xml packen. Dann kann man da " Uhr" hinschreiben oder so.